File: u3u2/Assets/Scripts/app/net/s2c/GCChannelExchange.cs

using System;
namespace app.net
{
/**
 * 通过码兑换奖励结果
 *
 * @author CodeGenerator, don't modify this file please.
 */
public class GCChannelExchange :BaseMessage
{
	/** 兑换结果，1成功，2失败 */
	private int result;

	public GCChannelExchange ()
	{
	}

	protected override void ReadImpl()
	{
	// 兑换结果，1成功，2失败
	int _result = ReadInt();


		this.result = _result;
	}
	
	protected override void WriteImpl()
    {
        return;
    }
	
	public override short GetMessageType() 
	{
		return (short)MessageType.GC_CHANNEL_EXCHANGE;
	}
	
	public override string getEventType()
	{
		return HumanGCHandler.GCChannelExchangeEvent;
	}
	

	public int getResult(){
		return result;
	}
		

}
}

File: server/zlj2019/game_server/src/com/imop/lj/gameserver/human/msg/GCChannelExchange.java
package com.imop.lj.gameserver.human.msg;

import com.imop.lj.core.msg.MessageType;
import com.imop.lj.gameserver.common.msg.GCMessage;
/**
 * 通过码兑换奖励结果
 *
 * @author CodeGenerator, don't modify this file please.
 */
public class GCChannelExchange extends GCMessage{
	
	/** 兑换结果，1成功，2失败 */
	private int result;

	public GCChannelExchange (){
	}
	
	public GCChannelExchange (
			int result ){
			this.result = result;
	}

	@Override
	protected boolean readImpl() {

	// 兑换结果，1成功，2失败
	int _result = readInteger();
	//end



		this.result = _result;
		return true;
	}
	
	@Override
	protected boolean writeImpl() {

	// 兑换结果，1成功，2失败
	writeInteger(result);


		return true;
	}
	
	@Override
	public short getType() {
		return MessageType.GC_CHANNEL_EXCHANGE;
	}
	
	@Override
	public String getTypeName() {
		return "GC_CHANNEL_EXCHANGE";
	}

	public int getResult(){
		return result;
	}
		
	public void setResult(int result){
		this.result = result;
	}
}